src/popup/router/components/Overview.vue

```diff
@@ -90,18 +109,36 @@ export default {
     ActionIcon,
     AensIcon,
     Avatar,
+    TransactionTag,
   },
   props: {
     title: { type: String, required: true },
+    txFunction: { type: String, default: null },
     sender: { type: Object, required: true },
     recipient: { type: Object, required: true },
   },
+  computed: {
+    isNewUI() {
+      return !!this.$route.meta.newUI;
+    },
+    getTxType() {
+      return this.$te(`transaction.dexType.${getDexTransactionTag[this.txFunction] || this.txFunction}`)
```

Comment:
```suggestion
      return this.$t(`transaction.dexType.${getDexTransactionTag[this.txFunction] || this.txFunction}`)
```

Reply:
It's not the same, `$te` checks whether translation exists - if we will proceed with your suggestion & there won't be exact translation we will end up with always being true which will cause some problems with display